README.md

```diff
@@ -281,6 +281,15 @@ var httpClient = new HttpClient(new HttpClientXRayTracingHandler(new HttpClientH
 httpClient.GetAsync(URL);
 ```
 
+Using the [HttpClientFactory](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) pattern
```

Comment:
I believe that the HttpClientFactory is available for only for .Net Core 2.1 and the `Microsoft.Extensions.Http` nuget package is required for it. Mentioning that here would be helpful for the users. Also just for a consistent formatting, please make the heading bold.